### ISSUE TITLE
Mapping share button on Xbox Series X|S controllers to screenshot

### ISSUE THREAD

**Issue by opencooper** [cathery/sys-con#314]
Firstly, thanks so much for such useful software. I have an Xbox Series X|S controller, which works fine, with screenshotting currently mapped to the sync button. However, compared to previous Xbox controllers, this one has an extra "Share" button which Steam for example uses to screenshot. You can see a visual of the controller [here](https://support.xbox.com/en-US/help/hardware-network/controller/get-to-know-your-xbox-series-x-s-controller).

Is there a way to expose this in the config file so it can be mapped?

**Comment by Zatheroger** [comment]
How did you manage to make the xbox series x controller work? Mine is detected but the buttons don't do anything...

**Comment by opencooper** [comment]
@Zatheroger It worked out of the box for me. First, I installed sys-con 0.6.4 via the HB App Store. Then I restarted my system. I use a third-party dock powered by the official Nintendo USB-C power brick. I plugged a (JSAUX brand) USB-A to USB-C cable into the back of the dock, and then the USB-C end into the XBox controller. The controller should vibrate to indicate that it has powered on. From there, everything worked, with all buttons properly mapped. (the only unmapped being the Share button, hence this Issue)

Make sure you restarted your system, and try another USB cable. Not sure if it would affect anything, but I would also make sure the controller isn't synced wirelessly to anything else, as that might cause the cable to power it only. Note, the description for sys-con says "[Switch FW 5.0.0+] [Atmosphère only]".

**Comment by Zatheroger** [comment]
Yeah I found the problem. The controller does not work on any of the 3 usb ports the dock (the Nintendo one) has, however it works flawlessly when I plug it to the Switch itself via the usb type-c port. Which means I have to play on handheld if I want to use my xbox controller...

**Comment by Zatheroger** [comment]
Ok so I checked the discussions and someone said to "Press the Xbox button so it starts blinking, then plug the controller into the switch" and it worked! I hope I can also map the share button to something on the switch.

**Comment by SaddyGR3** [comment]
I can confirm what Zatheroger said worked perfectly. Now I can use the Xbox Series X controller connected to the original dock
